source/_posts/front-commerce-2.2.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Release: Front-Commerce 2.2"
+date: 2020-11-18
+---
+
+Front-Commerce 2.2 has a heavy focus on Magento 1 features thanks to new clients migrating their Legacy Magento 1 shops on Front-Commerce. Read more about this version.
+
+<!-- more -->
+
+## Guest Checkout
+
+It is now possible to complete a Checkout in Front-Commerce without being logged. This is a frequently asked feature as it can really boost your sales. This also means that we've made all the changes necessary to the theme and that it will now be possible to add the Guest Checkout to Magento 2 with very few changes. Stay tuned!
+
+## More product types
+
+In Front-Commerce we already supported simple products and configurable products which are the most commonly used types in the Magento ecosystem. In Front-Commerce's Magento 1 integration it is now possible to use Bundle Products and Virtual Products.
+
+Virtual products are especially exciting because this means that it is now possible to remove completly the shipping step from the checkout if a customer only buys virtual items.
```

Comment:
```suggestion
Virtual products are especially exciting because this means that it is now possible to completely remove the shipping step from the checkout if a customer only buys virtual items.
```